Gems/ROS2/Code/Source/Spawner/ROS2SpawnerComponent.cpp

```diff
@@ -146,9 +154,14 @@ namespace ROS2
             PreSpawn(id, view, transform, spawnableName, spawnableNamespace);
         };
 
-        spawner->SpawnAllEntities(m_tickets.at(spawnableName), optionalArgs);
+        optionalArgs.m_completionCallback = [service_handle, header](auto id, auto view)
```

Comment:
Is this guaranteed to be called in each possible outcome of spawning? Is there an option to add a timeout?